docs/install/advanced-configuration.rst

```diff
@@ -48,7 +48,7 @@ Geotrek-admin will send emails:
 * to managers when a feedback report is created
 
 Email configuration takes place in ``/opt/geotrek-admin/var/conf/custom.py``, where you control
-recipients emails (``ADMINS``, ``MAIL_MANAGERS``) and email server configuration.
+recipients emails (``ADMINS``, ``MANAGERS``) and email server configuration.
```

Comment:
We assume that administrators will understand how to fill these parameters or will check examples in https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/settings/custom.py.dist ?